app/controllers/concerns/v1/profiles_controller/attributes.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module V1
+  class ProfilesController
+    # Helper methods related to profile attributes
+    module Attributes
```

Comment:
This new helper is thanks to rubocop's class length cop. It's copied from profiles_controller with the addition of policy-related attributes.